test/transformers/test_rms_norm.py

```diff
@@ -74,6 +74,7 @@ def forward(self, x):
         return output.type_as(x)
 
 
+@pytest.mark.flaky(reruns=3, reruns_delay=2)
```

Comment:
will this count as "pass" after all rerun "fails"?

Reply:
It will be counted as "FAILED".